ui/raidboss/data/02-arr/raid/t1.js

```diff
@@ -119,4 +119,71 @@
       },
     },
   ],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'replaceSync': {
+        'Ads': 'Abwehrsystem',
+        'Caduceus': 'Caduceus',
+        'Dark Matter Slime': 'Dunkelmaterien-Schleim',
+        'The Allagan megastructure': 'Allagische Superstruktur',
+      },
+      'replaceText': {
```

Comment:
I don't think it's worth adding all these empty unused sections.  <_<

Reply:
than i will remove them, if it makes you happy :)